services/question/README.md

```diff
@@ -350,11 +347,12 @@ This endpoint allows updating an existing question. Only the title, description,
 
 ### Responses:
 
-| Response Code               | Explanation                                    |
-|-----------------------------|------------------------------------------------|
-| 200 (OK)                    | Success, the question is updated successfully. |
-| 404 (Not Found)             | Question with the specified `id` not found.    |
-| 500 (Internal Server Error) | Unexpected error in the database or server.    |
+| Response Code               | Explanation                                                                        |
+|-----------------------------|------------------------------------------------------------------------------------|
+| 200 (OK)                    | Success, the question is updated successfully.                                     |
+| 404 (Bad Request)           | Invalid request body such as including `id` or duplicate `title` or `description`. |
```

Comment:
```suggestion
| 400 (Bad Request)           | Invalid request body such as including `id` or duplicate `title` or `description`. |
```
Think there's a typo here

Reply:
Yeap, thanks for noticing! I will be changing this :>